app/_data/docs_nav_contributing.yml

```
@@ -10,6 +9,10 @@
       url: /style-guide
     - text: Word choice and naming
       url: /word-choice
+    - text: Notes and other notices
```

```suggestion
    - text: Notes and other notices
```
How about "warnings", "note tags", "calling attention".

---

"Notes and warnings" ?

---

Though in the doc you used "notices" a lot, maybe we should just call this "Notices"